storybook/src/stories/helpers/data_samples.tsx

```diff
@@ -443,6 +443,9 @@ const resultTextTemplateJp = '表示結果 ([TOTAL_RESULTS]):';
 const clearEng = 'CLEAR ALL';
 const clearJp = 'すべてクリア';
 
+const jpMonths: string[] = ["1月", "2月", "3月", "4月", "5月", "6月", "7月", "8月", "9月", "10月", "11月", "12月"];
+const jpDays: string[] = ['日', '月', '火', '水', '木', '金', '土'];
```

Comment:
Have you check the new code with eslint? If lint is configured then please check and fix linting issues of all the new code which you have written.

Reply:
Done